images/singleuser/Dockerfile

```diff
@@ -141,6 +141,11 @@ RUN pip install --no-cache-dir git+https://github.com/yuvipanda/nbpawspublic@023
 
 RUN pip install --no-cache-dir git+https://github.com/yuvipanda/ipynb-paws@60c94e3
 
+# use custom css to hide clusters tab
+COPY hide_clusters_tab.css /home/paws/.jupyter/custom/custom.css
```

Comment:
COPY now allows you to have a ` --chown=tools.paws:tools.paws ` modifier and save a few layers.

Reply:
I change it to use the ``chown`` modifier. Is the Dockerfile fines now?